extensions/analyticsdx-vscode-templates/src/templateEditing.ts

```diff
@@ -367,15 +400,6 @@ export class TemplateDirEditing extends Disposable {
         providedCodeActionKinds: LayoutVariableCodeActionProvider.providedCodeActionKinds
       }),
 
-      // hookup quick fixes for variable tile keys in layout.json's
-      vscode.languages.registerCodeActionsProvider(
```

Comment:
This is now handled by the common FuzzyMatchCodeActionProvider registration above.